www/nginx/src/opnsense/scripts/nginx/setup.php

```diff
@@ -304,7 +304,7 @@ function find_ca($refid)
     if ((string)$tls_fingerprint->trusted == '1') {
         $ciphers = explode(':', (string)$tls_fingerprint->ciphers);
         if (!empty((string)$tls_fingerprint->curves)) {
-            $curves = explode(':', (string)$tls_fingerprint->ciphers);
+            $curves = explode(':', (string)$tls_fingerprint->curves);
```

Comment:
isn't this a separate bugfix that should be mentioned in the release notes?

Reply:
sure). fixed, thanks